src/share/streams/shr_dmodel_mod.F90

```diff
@@ -38,7 +38,7 @@ module shr_dmodel_mod
   end interface shr_dmodel_gsmapCreate
 
   interface shr_dmodel_mapSet; module procedure &
-      !shr_dmodel_mapSet_dest, &
+       !shr_dmodel_mapSet_dest!, &
```

Comment:
Does this comment out the line?  Why not just remove it?

Reply:
The original already had this line commented out  - not sure why the change was made, I can remove it or restore to the original.